PictureInPictureKotlin/app/src/main/java/com/example/android/pictureinpicture/MovieActivity.kt

```diff
@@ -56,50 +58,35 @@ class MediaSessionPlaybackActivity : AppCompatActivity() {
         private const val PLAYLIST_SIZE = 2
     }
 
-    private lateinit var session: MediaSessionCompat
-
-    /** The arguments to be used for Picture-in-Picture mode.  */
-    private val pictureInPictureParamsBuilder = PictureInPictureParams.Builder()
+    private lateinit var binding: MovieActivityBinding
 
-    /** This shows the video.  */
-    private lateinit var movieView: MovieView
+    private lateinit var session: MediaSessionCompat
 
-    /** The bottom half of the screen; hidden on landscape  */
-    private lateinit var scrollView: ScrollView
-
-    private val onClickListener = View.OnClickListener { view ->
-        when (view.id) {
-            R.id.pip -> minimize()
-        }
-    }
+    private val rect = Rect()
```

Comment:
nit: Consider a more meaningful name than `rect`? (Could it be a local variable?)

Reply:
Done